frontend/ios/Runner/AppDelegate.swift

```diff
@@ -7,7 +7,51 @@ import Flutter
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
+    clearITNRWStorage()
     GeneratedPluginRegistrant.register(with: self)
     return super.application(application, didFinishLaunchingWithOptions: launchOptions)
   }
+    
+    func clearITNRWStorage() {
+        let preferences = UserDefaults.standard
+        let key = "preferencesSQLFilterString"
```

Comment:
This is a string that was used by IT.NRW and that should never be used by us?
Can we document this here?